react/src/components/Forms/LanguageForm.jsx

```diff
@@ -1,25 +1,31 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { ContinueBtn } from '../Buttons/ContinueBtn';
-import { PreviousBtn } from '../Buttons/PreviousBtn';
 import { Logo } from '../Shared/Logo';
 import { LanguageSelection } from '../Shared/LanguageSelection';
+import { AuthContext } from '../../auth';
+
+export const LanguageForm = ({ action, userType }) => {
+  const context = useContext(AuthContext);
+  let header;
+
+  if (context.user.type === 'teacher' || userType === 'teacher') {
```

Comment:
Do we need to be checking both of these?

Reply:
Yes because when a user clicks to go to the next page i am not firing off a new get request to update the user object. So i have to check if the user object has the type (i.e they left midway through the sign up process) and I have to check the state (they didnt leave halfway through the process). I plan on going back to refactor if there is time and make it cleaner